datastore/db/bytebase/FertiScan/update_inspection_function.sql

```diff
@@ -375,6 +375,30 @@ END;
 $$ LANGUAGE plpgsql;
 
 
+-- Function to check if both text_content_fr and text_content_en are NULL, and skip insertion if true
+CREATE OR REPLACE FUNCTION "fertiscan_0.0.14".check_null_sub_label()
+RETURNS TRIGGER AS $$
+BEGIN
+    -- Check if both text_content_fr and text_content_en are NULL
+    IF NEW.text_content_fr IS NULL AND NEW.text_content_en IS NULL THEN
+        -- Raise a notice and skip the insertion by returning NULL
+        RAISE NOTICE 'Skipping insertion because both text_content_fr and text_content_en are NULL';
```

Comment:
I'd even make this an Exception, to make sure it doesn't happen. 
```suggestion
        RAISE Exception 'Skipping insertion because both text_content_fr and text_content_en are NULL';
```

Reply:
Done